.github/workflows/ci.yml

```
@@ -66,28 +66,8 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       with:
         command: version
-    - name: Writing bundle
+    - name: Building bundle
```

```suggestion
    - name: Writing bundle
```

This actually **writes** the bundle to disk. The bundle has already been _built_ and has been embedded in the library, we are just outputting it from the const here.